validator_client/src/lib.rs

```diff
@@ -611,6 +609,9 @@ impl<T: EthSpec> ProductionValidatorClient<T> {
             );
         }
 
+        // Wait until genesis has occurred.
+        wait_for_genesis(&self.beacon_nodes, self.genesis_time, &self.context).await?;
```

Comment:
We also start all the other services like the `block_service`/etc. I haven't had time to run this PR, but if someone (@antondlr) can confirm that these services run without errors prior to genesis I think we can merge.

Reply:
just pushed this to all the `holly` VCs; looking good:
https://caribou.spesi.io/app/r/s/FXRmX

Reply:
I see a `CRIT Failed to poll sync committee duties`